docroot/modules/custom/mass_content/js/session_orgs.js

```diff
@@ -5,79 +5,86 @@
               if ($.inArray(el, orgsFiltered) === -1) {
                 orgsFiltered.push(el);
               }
             });
           }
-          // Set Session start time for each user.
-          var sessionStart = sessionStorage.getItem('session_start');
-          if (sessionStart && sessionStart.length > 0) {
-            // Remove session_orgs from storage only if the hours difference
-            // is more than 1 hour.
-            if (diff_hours(new Date(), new Date(sessionStart)) >= 1) {
-              sessionStorage.setItem('session_start', new Date().toString());
-              var sessionOrgs = sessionStorage.getItem('session_orgs');
-              if (sessionOrgs && sessionOrgs.length > 0) {
-                if (orgsFiltered.length > 0) {
-                  // Set session_orgs value to storage.
-                  sessionStorage.setItem('session_orgs', orgsFiltered.join(','));
-                }
+
+          if ($("meta[name='mg_parent_org']").length > 0) {
+            // Get data from the metatag element.
+            parentOrgs = $("meta[name='mg_parent_org']").attr('content');
+            // Filter the array and keep only unique values.
+            var parentOrgsArr = parentOrgs.split(',');
+            $.each(parentOrgsArr, function (i, el) {
+              if ($.inArray(el, parentOrgsFiltered) === -1) {
+                parentOrgsFiltered.push(el);
               }
-            }
-          }
-          else {
-            // Set session_start value to storage.
-            sessionStorage.setItem('session_start', new Date().toString());
+            });
           }
 
-          var sessionOrgsExisting = sessionStorage.getItem('session_orgs');
-          if (sessionOrgsExisting && sessionOrgsExisting.length > 0) {
-            // Convert string to array to filter for unique values.
-            var existingValues = sessionOrgsExisting.split(',');
-            if (orgsFiltered.length > 0) {
-              // Combine existing and new data together in 1 array.
-              var resultValues = existingValues.concat(orgsFiltered);
-              var resultFiltered = [];
-              // Filter the array and keep only unique values.
-              $.each(resultValues, function (i, el) {
-                if ($.inArray(el, resultFiltered) === -1) {
-                  resultFiltered.push(el);
-                }
-              });
-              // Set session_orgs value to storage.
-              sessionStorage.setItem('session_orgs', resultFiltered.join(','));
-            }
-          }
-          else {
+          // Set Session start time for each user.
+          var sessionStart = sessionStorage.getItem('session_start');
+          // Reset session_start, session_orgs and session_parent_orgs from storage at sessionStart or if the hours difference is more than 1 hour.
+          if (!sessionStart || sessionStart.length <= 0 ||  diff_hours(new Date(), new Date(sessionStart)) >= 1) {
```

Comment:
@clairesunstudio I'm seeing this error at this line in our validation check: `Multiple spaces found before 'diff_hours' `. This should be the fix. Submitting.
```suggestion
          if (!sessionStart || sessionStart.length <= 0 || diff_hours(new Date(), new Date(sessionStart)) >= 1) {
```